pom.xml

```diff
@@ -16,7 +16,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.scm.id>github</project.scm.id>
     <doclint>none</doclint>
-    <driver.version>4.14.1</driver.version>
+    <driver.version>4.15.0</driver.version>
```

Comment:
where s this coming from?

Reply:
From v1. Updates other than DSE retained -- this included. Should not make a difference (I tried with both versions and DSE 6.8.30/.31 had same issue).